### PR TITLE
Fix note

### PR DESCRIPTION
**Pull request by xieyanker**
image

### CODE REVIEW

**Comment by xieyanker** [comment]
/assign @tengqm
https://deploy-preview-21173--kubernetes-io-master-staging.netlify.app/zh/docs/contribute/intermediate/#%E4%BD%BF%E7%94%A8%E6%9C%AC%E5%9C%B0%E4%BB%93%E5%BA%93

File: content/zh/docs/contribute/intermediate.md [hunk 1]
```diff
@@ -696,14 +696,15 @@ most up-to-date version of that branch.
...
-      -->不要在提交消息中引用 GitHub issue 或 PR(通过 ID 或 URL)。如果您这样做了，那么每当提交出现在新的Git 分支中时，就会导致该 issue 或 PR 获得通知。稍后，您可以在 GitHub UI 中链接 issues 并将请求拉到一起。
-      {{< /note >}}
+{{< note >}}
+<!--
+Do not reference a GitHub issue or pull request by ID or URL in the
+commit message. If you do, it will cause that issue or pull request to get
+a notification every time the commit shows up in a new Git branch. You can
+link issues and pull requests together later, in the GitHub UI.
+-->
+不要在提交消息中引用 GitHub issue 或 PR(通过 ID 或 URL)。如果您这样做了，那么每当提交出现在新的Git 分支中时，就会导致该 issue 或 PR 获得通知。稍后，您可以在 GitHub UI 中链接 issues 并将请求拉到一起。
```
Comment (xichengliudui) [inline content/zh/docs/contribute/intermediate.md:706]:
```suggestion
不要在提交消息中引用 GitHub issue 或 PR(通过 ID 或 URL)。如果您这样做了，那么每当提交出现在新的 Git 分支中时，就会导致该 issue 或 PR 获得通知。稍后，您可以在 GitHub UI 中链接 issues 并将请求拉到一起。
```

Comment (xieyanker) [reply content/zh/docs/contribute/intermediate.md]: @xichengliudui Thanks for your suggestion, done!

**Comment by tengqm** [comment]
/lgtm
/approve
Thanks.

**Pull request merged**